test/openAPI/features/client_create.feature

```diff
@@ -1,16 +1,130 @@
-Feature: API to add new open ID connect (OIDC) clients, it can be invoked by other modules which manage the relying parties/partners.
+@method=POST @endpoint=/client-mgmt/oidc-client
+Feature: API to add new open ID connect (OIDC) clients.
 
-  Create OIDC Client Endpoint.
+  @smoke 
+  Scenario: The new client is successfully added to the Open ID Connect (OIDC) smoke type test
+    Given The user wants to add the new client to the Open ID Connect (OIDC)
+    When User sends POST request with given "2011-10-05T14:48:00.000Z" as requestTime, "e-health-service" as clientId, "Health Service" as clientName, "bharath-gov" as relyingPartyId, "http://example.com" as logoUri, empty object as publicKey, "idbb:acr:generated-code" as authContextRefs, "name" as userClaims, "authorization_code" as grantTypes, "private_key_jwt" as clientAuthMethods
```

Comment:
redirectUris is missing to be mentioned here

Reply:
It's missing because this is smoke test and `redirectUris` in the API spec is not reqiured

Reply:
Spoke to the internal team and found that redirectUris is a required field, so created the below PR on this and waiting for review.
#36

Reply:
Ok, so I will also change the tests now and add redirectUri as required